src/task/tests/pullRequests/pullRequest.spec.ts

```diff
@@ -221,9 +227,22 @@ describe('pullRequest.ts', (): void => {
   })
 
   describe('getUpdatedTitle()', (): void => {
+    it('should return null when the inputs disables changing the title', async (): Promise<void> => {
+      // Arrange
+      when(inputs.changePrTitle).thenReturn(false)
+      const pullRequest: PullRequest = new PullRequest(instance(codeMetrics), instance(logger), instance(runnerInvoker), instance(inputs))
+
+      // Act
+      const result: string | null = await pullRequest.getUpdatedTitle('S✔ ◾ Title')
```

Comment:
It would be better to pass in "Title" here as this seems to be testing the case where a prefix already exists.